proto/nexus/v1beta1/query.proto

```diff
@@ -22,3 +22,21 @@ message LatestDepositAddressRequest {
 }
 
 message LatestDepositAddressResponse { string deposit_addr = 1; };
+
+// TransfersForChainRequest represents a message that queries the
+// transfers for the specified chain
+message TransfersForChainRequest {
+  string chain = 1;
+  string state = 2;
+}
+
+message TransfersForChainResponse {
+  repeated Transfer transfers = 1 [ (gogoproto.nullable) = false ];
+  int32 count = 2;
```

Comment:
do we really need an extra field @jack0son? You already get the count implicitly with the array